packages/core/crates/core-db/src/db.rs

```diff
@@ -51,6 +41,23 @@ impl<T: BinaryAsyncKVStorage> HoprCoreDbActions for CoreDb<T> {
         Ok(())
     }
 
+    async fn get_acknowledged_tickets(&self, filter: Option<ChannelEntry>) -> Result<Vec<AcknowledgedTicket>> {
```

Comment:
We should get rid of such object specific code in the general DB module and instead move to object specific modules at some point.

Reply:
Indeed, once core-ethereum is rewritten to use this, some of these callbacks will likely change and that's when the docs can come in. Now the parity is ~1:1 with the parallel TS code.

Reply:
This API can look significantly better given 2 details:
- the consumer is Rust (which it is not)
- the DB engine is more flexible (which levelDB is not)

But it will be improved already in Providence.